include/atari5200.h

```diff
@@ -89,5 +89,8 @@ extern void atr5200std_joy[];        /* referred to by joy_static_stddrv[] */
 */
 #define _bordercolor(color) 0
 
+extern void waitvsync (void);
+/* Wait for start of next frame */
```

Comment:
The other comments in this file are all above the line they refer to. (And maybe you want to take the opportunity to harmonize uppercase usage.)